### ISSUE TITLE
Missing ./dist/react-fullpage.js on v0.1.20

### ISSUE THREAD

**Issue by AbeEstrada** [alvarotrigo/react-fullpage#297]
### Description

I'm getting this error:

```
Module not found: Can't resolve '@fullpage/react-fullpage'
```

### Steps to reproduce it

```
import ReactFullpage from "@fullpage/react-fullpage";
```

I cloned the repo and the `build` script doesn't generate a `"./dist/react-fullpage.js"` file.

### Versions 

v0.1.20

**Comment by AbeEstrada** [comment]
@alvarotrigo the `CleanWebpackPlugin` is running twice, it removes the recently created `react-fullpage.js` on the second merge

**Comment by AbeEstrada** [comment]
Maybe this should fix it?

```js
plugins: [new CleanWebpackPlugin({ cleanStaleWebpackAssets: false })]
```

**Comment by alvarotrigo** [comment]
Awesome, that did the trick!

Fixed and merged into a new release 0.1.21.
